advanced/Scripts/query.sh

```diff
@@ -88,11 +61,41 @@ case "${options}" in
     *              ) domainQuery="${options}";;
 esac
 
+# convert the domain to lowercase
+domainQuery=$(echo "${domainQuery}" | tr '[:upper:]' '[:lower:]')
```

Comment:
```suggestion
domainQuery="${domainQuery,,}"
```

Reply:
Although it looks nicer - going forward we are trying to avoid bashisms (we're not strictly enforcing this at the moment), with the ultimate goal being posix compatibility and increased portability.

Reply:
Absolutely we are, which is why I said going forward.

I think the unwritten/unspoken rule at the moment is "If we can do it without a bashism, then that is the preference". At some point we will sit down and convert entire scripts, but as you can imagine that is a daunting task and there is only so much free time!

utils.sh and version.sh are already posix compliant (as is PADD)